backend/gitSound.py

```diff
@@ -214,6 +214,27 @@ def removeSongFromPlaylist(self, pid, songid):
         repo.create_commit("HEAD", self.author, self.comitter,
                            "Removed " + songid, tree, [repo.head.target])
 
+    def songLookup(self, name=None, artist=None, limit=1):
```

Comment:
Not yet, but the Spotify API is weird about querying a track with an artist at the same time, so I figured later I would add functionality where if the person enters an artist along with a track name, then it checks for that artist after querying all the songs with that name. For now I just limited the results to return one song though so it isn't being used.